dashboard/app/graphs.go

```diff
@@ -176,20 +203,23 @@ func loadGraphBugs(c context.Context, ns string) ([]*Bug, error) {
 				continue
```

Comment:
For the total number of bugs found per month, I wonder if we're better off only counting the ones we have sent to the LKML or the ones we've released on our website (i.e. also including https://groups.google.com/g/syzkaller-upstream-moderation).